### ISSUE TITLE
MEDIUMTEXT field dumps as NULL

### ISSUE THREAD

**Issue by ghost** [sequelpro/sequelpro#1769]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=1769
Reporter:j...@jvmyers.com Date:2013-07-31 11:28:18 Status:WontFix Closed:2013-10-24

<b>What steps will reproduce the problem?</b>
1. create table with single field &quot;all_x&quot; of string type MEDIUMTEXT
2. create single table entry with &quot;all_x&quot; field containing &gt; 65,535 (216 – 1) characters, e.g., 65700 &quot;x&quot; characters
3. dump table

What is the expected output? the dump of &quot;all_x&quot; field should contain 65,700 &quot;x&quot; characters
What do you see instead? &quot;all_x&quot; field dumps as &quot;NULL&quot;

What version of Sequel Pro are you using? 1.0.2 build 4096
What version of OS X are you using? 10.6.8
What version of MySQL are you connecting to on the server? 5.0.45

<b>Please provide any additional information below.</b>

Love Sequel-Pro on my Macs!  I support 11 websites, 2 of which have 'relatively' large tables.  However, those tables have, until just very recently, required no text fields of size larger than TEXT, i.e., 65,535 (216 – 1) characters.  A couple of weeks ago, I finally had to increase the size of one field in one table to MEDIUMTEXT.  Although I make nightly backup dumps of my MySQL databases, it was not until today that I needed to access a dump made after the change to MEDIUMTEXT.  Upon loading this table into my dev DB, I quickly noticed that the MEDIUMTEXT field in SOME (but certainly not ALL) of the table entries was NULL.  A quick check revealed that the dumped value was &quot;NULL&quot; ONLY if the field actually contained &gt; 65,535 (216 – 1) characters -- all other table entries with smaller content in this field dumped just fine!!

I'll be happy to cooperate further if requested.
